Packs/CommonScripts/ReleaseNotes/1_11_21.md

```diff
@@ -0,0 +1,5 @@
+
+#### Scripts
+##### New: ShowIncidentIndicators
+- This script is used to display the indicators of an incident in an incident field of type Array. It can be used to select indicators from the incident in order to later perform some actions, like tagging the indicators for blocking via EDL.
```

Comment:
```suggestion
This script is used to display the indicators of an incident in an incident field of type Array. It can be used to select indicators from the incident in order to later perform some actions, like tagging the indicators for blocking via EDL.
```